Solutions/Windows Security Events/Hunting Queries/uncommon_processes.yaml

```diff
@@ -1,6 +1,8 @@
 id: 2ff4b10c-7056-4898-83fd-774104189fd5
 name: Uncommon processes - bottom 5%
 description: |
+  'Query highlights first-time rare processes, best over 3+ days, to flag new, potentially unauthorized binaries in stable environments for attack detection.'
```

Comment:
Query highlights uncommon, rare process runs, to flag new potentially unauthorized binaries in stable environments for potential attack detection.